cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -3668,5 +3672,101 @@
     {
       "$ref": "#/$defs/reporting_config"
     }
-  ]
+  ],
+  "properties": {
```

Comment:
Now that we have an "empty" top-level "properties" key present `cloudinit.config.schema._get_property_doc` needs to grow smarts to not document completely "empty/undefined properties" for modules which have no specific subschema.

Reply:
Do you have a problematic example? I'm not sure I understand.

Reply:
Sorry, I hadn't explained this context. We don't want our schema docs to reprsent this set of empty top-level properties when a given module does not actually have a subschema defined.
The way we avoid producing invalid top-level docs for a module which defined no schema is to avoid generating docs with _get_properties_doc when we see no specialized subschema.

This [property doc "skip" is done in the following commit](https://github.com/canonical/cloud-init/commit/ad5ac9383d1bb321d571060b796e0f5b91142d2e)